README.md

```diff
@@ -125,3 +125,8 @@ export function unmount(props) {
 
 ## API
 See [single-spa api](/docs/single-spa-api.md) and [child application api](/docs/child-applications.md#child-application-lifecycle).
+
+## Who's Using This?
+Please submit a P.R. to this section if you start using single-spa.
+
+- [CanopyTax](www.canopytax.com)
```

Comment:
I also know of DealerSocket and Beamery who are using this in prod. Should we add them to this list? Or maybe get their permission, first

Reply:
We should reach out to them, and let them add themselves.
Would probably be incorrect to add others.